pythran/pythonic/__builtin__/list/append.hpp

```diff
@@ -23,6 +23,12 @@ namespace pythonic
         seq.push_back(std::forward<F>(value));
         return __builtin__::None;
       }
+      template <class T, class F>
+      types::none_type append(types::list<T> &&seq, F &&value)
+      {
+        seq.push_back(std::forward<F>(value));
+        return __builtin__::None;
+      }
```

Comment:
Nice to have this feature. For information, I think some function doesn't have a correct implementation for rvalue (do nothing as we didn't plan this).